explorer/app/apis/ncpi-catalog-dug/common/entities.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Response from Dug API call
+ */
+
+export interface DugVariableResponse {
+  results: DugVariableResponse[];
```

Comment:
This is looking close! 

I think I actually missed a couple of things in our related conversation on [Slack](https://clevercanary.slack.com/archives/C03S7KKE7S5/p1659504427921249). The Dug response has the following format:

```
{
  status: "success",
  result: {
    DbGaP: [...]
  },
  ...
};
```

So I think we can:

1. Update `results` to `result`.
2. Update `result` type to an intermediate object (`DugDbGapResponse`?); it will need a single key `DBGaP` with a type of `DugVariableResponseElement[]`.

Let me know if this isn't clear!

Reply:
Sorry, I meant to have the result be DugVariableResponseElement just completely missed it thanks!